Applications/FileIO/SWMM/SWMMInterface.cpp

```diff
@@ -1151,14 +1148,14 @@ std::string SwmmInterface::getArrayName(SwmmObject obj_type, std::size_t var_idx
         if (var_idx < n_obj_params[1])
             return node_vars[var_idx];
         if (var_idx < n_obj_params[1]+n_pollutants)
-            return _pollutant_names[var_idx-n_obj_params[1]];
+            return std::string("Node_" + _pollutant_names[var_idx-n_obj_params[1]]);
```

Comment:
Does this *really* make a difference? I would assume that a string is constructed implicitely either way because the method cannot return multiple string fragments. If this can be avoided somehow I'm sure the compiler will do so in the optimization process.

And even if none of this happens it'll simply make no noticable difference in runtime. This is about returning 10-20 names in a process that reads/writes hundreds of megabytes.

It's just that the explicit statement of a string construction makes this more readable in my opinion.